index.js

```diff
@@ -62,6 +62,6 @@ module.exports = {
     "jest/no-identical-title": "error",
     "jest/prefer-to-have-length": "warn",
     "jest/valid-expect": "error",
-    "no-jest-import": "error"
+    "no-jest-import": "0"
```

Comment:
shouldn't this be `"no-jest-import": "0"`
```suggestion
    "no-jest-import": 0
```